### PR TITLE
selinux: update to latest release

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Merge PRs into one to test it against CI.

### CODE REVIEW

**Comment by robimarko** [comment]
Its weird to see it fail, compiles locally just fine

File: package/libs/libselinux/Makefile [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
-PKG_VERSION:=3.3
-PKG_RELEASE:=2
+PKG_VERSION:=3.5
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://github.com/SELinuxProject/selinux/releases/download/$(PKG_VERSION)
-PKG_HASH:=acfdee27633d2496508c28727c3d41d3748076f66d42fccde2e6b9f3463a7057
+PKG_HASH:=9a3a3705ac13a2ccca2de6d652b6356fead10f36fb33115c185c5ccdf29eec19
 HOST_BUILD_DEPENDS:=libsepol/host pcre/host
```
Comment (lovehackintosh) [inline package/libs/libselinux/Makefile:15]: Why this package defined HOST_BUILD_DEPENDS twice?
And pcre/host should be pcre2/host

**Comment by PolynomialDivision** [comment]
> Though there is a bunch of packages depending on `pcre` and not `pcre2`

But they are only located in the openwrt/packages?

Yeah, it's weird. Everything builds also fine on my system. I will now test again with docker container. Let's see why it fails.

**Review by misaka36 — COMMENTED** [review]
Commit message has a typo error, doulbe should be double

**Comment by PolynomialDivision** [comment]
Builds now fine! :) So we can move pcre to openwrt/packages and than merge this here.

**Comment by ynezz** [comment]
>  We can't move pcre to openwrt/packages because some packages of telephony are using it.

Yes, you can do that and should do that. If you for example look at asterisk dependencies, you'll find out, that you won't be able to built it without packages feed anyway, its not free standing feed.

**Comment by PolynomialDivision** [comment]
merged to master. @lovehackintosh thanks! :)

**Comment by PolynomialDivision** [comment]
@lovehackintosh I think you forgot to update selinux-policy?

**Comment by lovehackintosh** [comment]
> @lovehackintosh I think you forgot to update selinux-policy?

#12743

**Pull request closed without merging**